azurerm/utils/common_marshal.go

```diff
@@ -12,6 +12,15 @@ func ExpandStringSlice(input []interface{}) *[]string {
 	return &result
 }
 
+func ExpandMapStringPtrString(input map[string]interface{}) map[string]*string {
+	result := make(map[string]*string)
+	for k, v := range input {
+		s := v.(string)
+		result[k] = &s
```

Comment:
I think we can consolidate this just a bit
```suggestion
		result[k] = utils.String(v.(string))
```